doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -191,6 +191,14 @@ Bluetooth LE
 
 .. rst-class:: v2-9-0-nRF54H20-rc1 v2-9-0 v2-8-0 v2-7-0 v2-6-2 v2-6-1 v2-6-0
 
+DRGN-24352: hci_ipc and ipc_radio samples do not support stream flow control and drop bytes
+  With the deferred disconnection complete generation added in v2-6-0, we see in cases of CONFIG_BT_HCI_ACL_FLOW_CONTROL=y, and the CONFIG_BT_BUF_CMD_TX_COUNT  < (CONFIG_BT_BUF_ACL_RX_COUNT + 1), we may drop HCI command packets.
```

Comment:
```suggestion
  With the deferred disconnection complete generation added in the |NCS| v2.6.0, there are cases where the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option has value ``y``, and the value of :kconfig:option:`CONFIG_BT_BUF_CMD_TX_COUNT` is smaller than (:kconfig:option:`CONFIG_BT_BUF_ACL_RX_COUNT` + 1).
 This might result in dropped HCI command packets.
```